STYLE.md

```diff
@@ -0,0 +1,52 @@
+# Distributed Tracing Working Group Style Guide
+
+This document describes the editorial style preferred by the W3C Distributed Tracing Working Group.
+Where possible and appropriate, the rules in this document should be followed when making pull requests to repositories maintained by the working group.
+If a rule is not appropriate or possible in context, it may be acceptable to break the rule with the consent of the working group members reviewing a pull request.
+If it is not obvious why a rule is broken, a comment should be made in the document describing the reason.
```

Comment:
Not sure about that part. IIRC comments in markdown are possible, but somewhat awkward. For example, different markdown processors handle them differently. That might not be a problem for us, though.

Another alternative would be to recommend providing the reason in the commit comment instead of in the document?

(Note: If we decide on something else here, we would also want to update [CONTRIBUTING.md line 48](https://github.com/w3c/distributed-tracing-wg/pull/49/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R48))

Reply:
HTML comments can be done in markdown `<!-- comment here -->`. This shouldn't be a regular occurrence so I would think it shouldn't be too much of a burden?